source/reference/minio-mc/mc-license-register.rst

```diff
@@ -56,6 +82,11 @@ Register a Deployment Without Direct Internet Access
 
 Register a MinIO deployment at alias ``minio3`` on SUBNET that does not have direct Internet access due to a firewall, airgap, or the like.
 
+.. versionchanged:: mc RELEASE.2022-07-29T19-17-16Z
+
+   The current airgap registration process works with MinIO Client version ``RELEASE.2022-07-29T19-17-16Z`` or later.
```

Comment:
Perhaps drop `current` here? How likely are we to make older versions work in the future?

Reply:
Agree on dropping it. Our docs always target current release anyway.